test/test_torch.py

```diff
@@ -16462,23 +16471,24 @@ def test_addmm(self, device):
             self.assertEqual(res1, res2, atol=prec, rtol=0)
 
     @dtypes(torch.float, torch.double)
-    @dtypesIfCUDA(*([torch.float, torch.double] +
-                    ([] if TEST_WITH_ROCM else torch.testing.get_all_complex_dtypes())))
+    @dtypesIfCUDA(*((torch.float, torch.double, tfloat32) +
```

Comment:
Do you really need this approach and the previous decorator? It seems that with the recommended changes to the decorator you don't need to mimic new dtypes?

Reply:
fixed